modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/identity/governance/v1/IdentityGovernanceSuccessTest.java

```diff
@@ -142,8 +142,8 @@ public void testGetGovernanceConnectors() {
                     equalTo(category.getValue().getName()));
 
             validatableResponse.body("find{ it.id == '" + category.getKey() + "' }.self",
-                    equalTo("/t/" + tenant + "/api/server/v1" + IDENTITY_GOVERNANCE_ENDPOINT_URI + "/" +
-                            category.getValue().getId()));
+                    equalTo(getTenantQualifiedURLWithoutHostName("/api/server/v1" +
```

Comment:
shall we rename the method `getTenantQualifiedURLWithoutHostName` to something like `getRelativeTenantedURL' or `getTenantedRelativePath`?